cloud/linode/loadbalancers.go

```diff
@@ -50,6 +50,8 @@ const (
 
 	annLinodeHostnameOnlyIngress = "service.beta.kubernetes.io/linode-loadbalancer-hostname-only-ingress"
 	annLinodeLoadBalancerTags    = "service.beta.kubernetes.io/linode-loadbalancer-tags"
+
+	annLinodeNodePrivateIP = "node.k8s.linode.com/private-ip"
```

Comment:
based on all the other annotations should we do `service.beta.kubernetes.io/linode-private-ip`? be easier to add to docs too which you should do as a part of this pr either way: https://github.com/linode/linode-cloud-controller-manager#annotations

Reply:
This is the bikeshed I was asking for!

My reason for not doing that is this is not a service and in general, k8s reserves `kubernetes.io` for it's own purposes (`service.beta.kubernetes.io` seems to be an outlier).

> The kubernetes.io/ and k8s.io/ prefixes are reserved for Kubernetes core components.

https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/

Reply:
Examples of the pattern I followed:

AWS
https://github.com/search?q=%22k8s.amazonaws.com%22&type=code

LKE
https://github.com/search?q=%22lke.linode.com%22&type=code

Reply:
Added docs to the readme (ideally this will be automatic as we can pull this information from the API or Metadata as well, but that's a bigger diff because we don't currently PATCH Nodes with custom fields).

Reply:
AFAIK, `service.beta.kubernetes.io` is a special case, many cloud providers use it for LoadBalancer Service configuration. 

* I don't think it makes sense to use that annotation prefix for this case, on Node resources.
* I don't think there is strong pressure to change it for configuring LoadBalancer Services.

We have existing `linode.com` examples:

* [`topology.linode.com/`](https://github.com/search?q=org%3Alinode+%22topology.linode.com%2F%22+language%3AGo&type=code) is a reusable annotation that would logically apply to multiple resource types (nodes, services, volumes, obj, etc).
* [`lke.linode.com/`](https://github.com/search?q=%22lke.linode.com%22&type=code) applies to a specific product, mostly just clusters.

I actually don't see anything that needs to migrate to `k8s.linode.com`.

The `k8s` in `k8s.linode.com` is to signify that the annotation relates to a specific Kubernetes resource (`node.k8s.linode.com`). We can drop that too if that's preferable, we could use `topology.linode.com/private-ip` or similar, that seems fine.